apps/network_fs/server/connection_manager.cc

```diff
@@ -0,0 +1,374 @@
+#include "connection_manager.hh"
+#include "io.hh"
+
+#include <seastar/core/do_with.hh>
+#include <seastar/core/future-util.hh>
+#include <seastar/core/iostream.hh>
+#include <seastar/core/reactor.hh>
+#include <seastar/core/temporary_buffer.hh>
+
+#include <filesystem>
+#include <iostream>
+#include <vector>
+
+using namespace seastar;
+
+namespace fs = std::filesystem;
+using std::vector;
+using std::cerr;
+using std::endl;
+
+namespace {
+
+constexpr uint32_t alignment = 4096;
+
+future<> general_handler(output_stream<char>& output, std::exception_ptr e) {
+    int code = -1;
+    try {
+        std::rethrow_exception(e);
+    } catch(fs::filesystem_error& fe) {
+        cerr << "A filesystem exception: " << fe.code().message() << endl;
+        code = -fe.code().value();
+    } catch(...) {
+        cerr << "An unknown exception" << endl;
```

Comment:
`std::exception_ptr` is printable in seastar, so it's better to print it in order to see what happens:

```cpp
#include <seastar/util/log.hh>
logger fslogger("fs_server");

...

catch (...) {
    fslogger.warn("Unknown exception: {}", std::current_exception());
}
```